src/API/Management/Jobs.php

```diff
@@ -77,6 +85,16 @@ public function sendVerificationEmail($user_id, array $params = [])
             $body['client_id'] = $params['client_id'];
         }
 
+        if (! empty( $params['identity'] )) {
+            if ( empty( $params['identity']['user_id']) || ! is_string($params['identity']['user_id']) ) {
```

Comment:
not a requirement for this PR, but would be good if this could be extracted to a helper method that asserts params

Reply:
I agree, but per my other comments I don't know of a good way to do that without exposing the logic publicly. It could be something we improve when we get a PHP engineer on the team, or if we come up with a good solution before then.